packages/cli/src/commands/releasegold/locked-gold.ts

```diff
@@ -0,0 +1,95 @@
+import { flags } from '@oclif/command'
+import BigNumber from 'bignumber.js'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { Flags } from '../../utils/command'
+import { ReleaseGoldCommand } from './release-gold'
+
+export default class LockedGold extends ReleaseGoldCommand {
+  static description =
+    'Perform actions [lock, unlock, relock, withdraw] on the gold held in the given contract.'
```

Comment:
Is this just about locked gold? Perhaps `Perform actions [lock, unlock, relock, withdraw] on Celo Gold that has been locked via the provided ReleaseGold contract.`